practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,59 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+
+  useEffect(() => {
+    // Sử dụng fetch để fetch dữ liệu từ API
```

Comment:
please don't use vietnames in your code

Reply:
I updated
![image](https://github.com/datdragon777/React-Training/assets/74806826/799551ba-f24b-43e8-8759-83a8d37306f2)